Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
+  version "9.8"
+  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
+
+  depends_on "reflex"
```

Comment:
You have to push it to your own fork:

```
git remote add timbck2 https://github.com/timbck2/homebrew.git
git push -u timbck2 adding_vile
```

(I used `timbck2` as a name example, you can use something else like `myfork` for example).